frontend/src/booking/DetailsContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useHistory } from "react-router";
+import style from "./BookingPage.module.css";
+import changePath from "../general/helperFunctions";
+import messages from "../general/textHolder";
+
+const detailMessages = messages.details;
+const DetailsContainer = () => {
+  const history = useHistory();
+
+  return (
+    <div className={style.contentContainer}>
+      {/* Input fields go here */}
```

Comment:
Not too sure how useful this comment is

Reply:
Yeah sorry, it was more of a TODO for other members